package.json

```diff
@@ -67,14 +67,16 @@
     "postcss-loader": "4.2.0",
     "progress-stream": "2.0.0",
     "qrcode.react": "1.0.0",
+    "rc-rate": "2.9.1",
     "react": "17.0.1",
     "react-accessible-accordion": "3.3.4",
     "react-circular-progressbar": "2.0.4",
     "react-dom": "17.0.1",
     "react-hot-loader": "4.13.0",
     "react-modal": "3.11.1",
-    "react-range": "1.8.8",
     "react-number-format": "4.6.0",
+    "react-range": "1.8.8",
+    "react-ratings-declarative": "3.4.1",
```

Comment:
Can we use `rc-rate` instead of `react-ratings-declarative`? It seems `react-ratings-declarative` module hasn't updated since 3 years ago.

Reply:
@ngvtuan , I used rc-rate but it is pretty much different with our design. If react-ratings-declarative doesn't meet our requirements, I will change it down the road.